security/Authentication.rst

```diff
@@ -24,9 +24,9 @@ for that user, and these groups can have permissions granted to them in Kylo.
 Built-In Pluggable Authentication Profiles
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Kylo comes with some pre-built authentications configuration that may be
-activated by adding the approperiate Spring profiles to the UI and server
-configuration `application.properties` files.  By default, whenever any of these profiles
+Kylo comes with some pre-built authentication configurations that may be
+activated by adding the appropriate Spring profiles to the UI and server
+configuration `application.properties` file.  By default, whenever any of these profiles
```

Comment:
This should be 'files' because there is more than one (both kylo-ui and kylo-services).